docs/keystone-auth.md

```diff
@@ -0,0 +1,38 @@
+An authentication backend for Openstack's Keystone is maintained in this
+source tree as `haas.ext.auth.keystone`. This document describes its
+usage.
+
```

Comment:
Similar to my question in `ci/keystone/keystone.sh`, would it be prudent to claim to support only certain versions of keystone?